catalog.bom

```diff
@@ -138,102 +118,407 @@ brooklyn.catalog:
             RHELREPO: $brooklyn:config("mongodb.rhelrepo")
             PACKAGES: $brooklyn:config("mongodb.packagelist")
             MONGOPORT: $brooklyn:config("mongodb.port")
```

Comment:
My below error can be fixed by changing `$brooklyn:config("mongodb.port")` to `$brooklyn:attributeWhenReady("mongodb.port")` because a sensor is defined for this value with the proper format.